ci/check-documentation.py

```diff
@@ -78,6 +78,11 @@ def complain(message):
     omz_github_url = 'https://github.com/openvinotoolkit/open_model_zoo/'
 
     for md_path in sorted(all_md_files):
+
+        # skip the checks for Python Model API package
+        if 'model_api' in str(md_path):
+            continue
```

Comment:
Why is it skipped?

Reply:
Because CI fails with such checks. Only absolute paths to OMZ can be added in this documentation, since the package might be distributed from openvino-dev and used outside the OMZ

Reply:
Does that mean the file is not included to online docs?
> Only absolute paths to OMZ can be added in this documentation, since the package might be distributed from openvino-dev and used outside the OMZ

Can you explain how the distribution way of the package affects presence of absolute paths and why is it about absolute paths?

Reply:
> Can you explain how the distribution way of the package affects presence of absolute paths and why is it about absolute paths?  

Is it worth the documentation to contain the relative paths, if the package might be used in different repos? If someone opens the link to the demo in the documentation, it will be failed. So the relative paths would work only for online documentation

To avoid this I moved the list of supported demos from the Model API documentation to demos/readme.md